frontend/src/scenes/events/EventsTable.js

```diff
@@ -155,26 +155,36 @@ export class EventsTable extends Component {
                         {event.person}
                     </Link>
                 </td>
-                {params.map(param => (
-                    <td key={param} title={event.properties[param]}>
-                        <FilterLink
-                            property={param}
-                            value={event.properties[param]}
-                            filters={properties}
-                            onClick={(key, value) =>
-                                this.setState(
-                                    {
-                                        properties: {
-                                            ...properties,
-                                            [key]: value,
+                {params.map(paramRequest => {
+                    let param = paramRequest
```

Comment:
Why this variable assignment?

Reply:
It seemed the least-ugly option to do what was needed. `param` will be `$current_url` or `$lib`... and I didn't feel like adding `param === 'current_url' ? event.properties[param] || event.properties['$screen'] : event.properties[param]` or equivalent code to all the params passed to `<FilterLink>` and `<td>`. Thus this assignment.

Reply:
I'm also allergic to redefining params passed in via functions.

```
function bla (foo) {
  foo = notFoo
}
```
... is something that looks like bad code

Reply:
Sure, makes sense to me